### ISSUE TITLE
Trailing comma in ignore lists prevents app from opening

### ISSUE THREAD

**Issue by privateemu** [trembon/switch-library-manager#53]
I found a small issue where a trailing comma in the ignore lists would prevent the app from opening. For example I had, in effect:

```
 "ignore_update_title_ids": [
  "01234567890ABCDE",
  "01234567890ABCDF",
  "01234567890ABCEE",
]
```

As I forgot to clean it up myself and the app would not open. When I noticed the issue and fixed it to:

```
 "ignore_update_title_ids": [
  "01234567890ABCDE",
  "01234567890ABCDF",
  "01234567890ABCEE"
]
```

It worked fine. Minor issue in the grand scheme of things, but wanted to call it out.

**Comment by trembon** [comment]
As the file is in the json format and the json format does not allow trailing commas normally.

But i can check if the parsing library allows this by any flag.

**Comment by privateemu** [comment]
That makes sense, or a clean up of the array before loading the json, or just leaving this here for anyone else who is debugging crashing and happens to see this and it helps.

**Comment by trembon** [comment]
after reading and checking the documentation, to support this a whole different json parser would need to be implemented then the one in go as trailing commas are not supported by json, so this is not something that will be implemented. sorry.

**Comment by privateemu** [comment]
Fair enough, I appreciate you looking into it! Thanks for all the hard work you do to keep this project going!
